samples/word/25-paragraph/search.yaml

```diff
@@ -29,15 +30,17 @@ script:
         }
 
         async function wildcardSearch() {
+          // Do a wildcard search and highlight matches in the document.
           await Word.run(async (context) => {
-            // Check out how wildcard expression are built, also use the second parameter of the search method to include search modes
-            // (i.e. use wildcards).
+            // Check out how wildcard expressions are built and use the second parameter of the
+            // search method to include applicable search modes.
```

Comment:
This comment seems out of place. It doesn't follow the usual pattern of explaining the following code, but instead seems to be promoting the sample.